pagopa/io-app-design-system
# [IOAPPFD0-179] Add the first iteration of the automatic process to generate new `Icon...tsx` and `Pictogram…tsx` files.

## Short description
This PR adds the new `generateNewIcons.js` and `generateNewPictograms.js` processes to greatly reduce the delivery time of new icons and pictograms. Starting from the exported SVG files from Figma, the new process automatically generates the corresponding `Icon...tsx` or `Pictogram…tsx` files with all the changes that used to require manual correction.

> [!warning]
> This is the first iteration of the script. A testing period is required before it can be considered bulletproof. 

## List of changes proposed in this pull request
- Add the new `generateNewIcons` process
- Add the new `generateNewPictograms` process
- Add new icons and pictograms generated through the processes
- Add the new `timestamp.txt` file that will be updated each time the script is run.

## How to test
1. Go to the `src/components/icons` folder
2. Launch the following command: `$ node generateNewIcons.js`
3. If you want, try to add the new generated icons to the `Icon.tsx` file and check the **Icons** page in the example app.

or alternatively:

1. Go to the `src/components/pictograms` folder
2. Launch the following command: `$ node generateNewPictograms.js`
3. If you want, try to add the new generated icons to the `Pictogram.tsx` file and check the **Pictograms** page in the example app.
